packages/blueprints-integration/src/api/showStyle.ts

```diff
@@ -169,6 +169,17 @@ export interface ShowStyleBlueprintManifest<TRawConfig = IBlueprintConfig, TProc
 		coreConfig: BlueprintConfigCoreConfig
 	) => TProcessedConfig
 
+	/**
+	 * Validate the blueprint config passed to this blueprint according to the API schema, returning a list of messages to display to the user.
+	 */
+	validateConfigFromAPI?: (context: ICommonContext, apiConfig: object) => Array<IConfigMessage>
+
+	/** transform API blueprint config to the database format */
+	blueprintConfigFromAPI?: (context: ICommonContext, config: object) => TRawConfig
```

Comment:
I don't have strong opinions on this one, but since I know there has been discussions regarding internal vs external APIs before, I think the addition of this makes sense. It would be good to add to the description of this that `If not defined, the config will be passed through as a TRawConfig`, though.

Reply:
Comments have been updated